ROADMAP.md

```diff
@@ -1,8 +1,8 @@
  ## CI/CD: 
- - CD: Trigger releases on tags.
+ - ~~CD: Trigger releases on tags.~~ (#416)
    - **What?** Add CD capabilities to trigger a release when a tag is pushed.
    - **Why?** Save maintainer manual effort on releases. Increase cadence of releases.
- - Combining the Python and JS files into one installable package
+ - ~~Combining the Python and JS files into one installable package~~ (#415)
```

Comment:
nitpick but lets just link the ROADMAP item to the relevant tracking issue or PR and not strikethrough it out. When its actually released we can add the release number which contains the feature

Reply:
Ok, let's do that.